internal/blockchain/utxocache.go

```diff
@@ -451,15 +451,15 @@ func (c *UtxoCache) Commit(view *UtxoViewpoint) error {
 			continue
 		}
 
```

Comment:
there are a couple of typos in the commit message here: 
- s / `... It accomplishes this by introdcuing an ...`/ `It accomplishes this by introducing an ...`
- s / `... the non-obvious, and heretofore undocumented, ...`, / `the non-obvious, and therefore undocumented, ...`

Reply:
Fixed the first case.

The second one is correct.  [heretofore](https://www.merriam-webster.com/dictionary/heretofore) means "up to this time".